### DISCUSSION TITLE
Make navigation on docs fixed while scrolling

### DISCUSSION THREAD

**Discussion by drewg2009** [laravel/framework#46409] [Ideas]
The laravel docs are very long and I always find myself scrolling all the way back to the top to navigate elsewhere. 

Could the navigation for other pages be fixed so that when I scroll i can always see it?

**Comment by KentuckySato** [comment]
Good idea @drewg2009 

I suggest a button "Back to top" to improve user experience. It will be great, especially if the doc is very long like this page [Getting Started](https://laravel.com/docs/10.x/eloquent)

**Reply by drewg2009** [reply]
Yes I agree, most of the pages are long and it gets annoying to scroll back to the top each time

**Reply by Angel5a** [reply]
Fixed navigation would be better. Sometimes want to open other page in new tab and stay on the current position on the current page.

We need both of these solutions.
- Back to top - to quickly jump to the navigation of this page.
- Fixed navigation - to quickly jump to other pages.

We have a narrow column for docs itself, and a bunch of free space by sides. We can afford a fixed navigation for sure.
